dactyl-config.yml

```diff
@@ -1618,9 +1618,60 @@ pages:
 #             - en
 #             - ja
 
-    -   name: Manage the rippled Server
-        html: manage-the-rippled-server.html
+        # TODO: translate title & blurb
+    -   name: Interoperability - EVM Sidechain
+        html: evm-sidechains.html
         parent: tutorials.html
+        blurb: Learn how to interact with the EVM Sidechain Devnet.
+        template: pagetype-category.html.jinja
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/get-started-evm-sidechain.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/connect-metamask-to-xrpl-evm-sidechain.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/join-evm-sidechain-devnet.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/evm-sidechain-validator-security.md
+        targets:
+            - en
+            - ja
+
+        # TODO: translate
+    -   md: tutorials/interoperability/evm-sidechain-run-a-validator-node.md
+        targets:
+            - en
+            - ja
```

Comment:
Looks like this work might need to be updated to reflect the EVM stuff being moved off the site—you don't want to re-add the files here, I suspect.